dist/tools/uncrustify/uncrustify.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I'd prefer one script with either "--check" or inplace editing. what do you think?

Reply:
yes, sounds reasonable. I will push a fix of this.

Reply:
addressed!